src/main/java/io/debezium/connector/vitess/VitessType.java

```diff
@@ -14,10 +19,17 @@
     private final String name;
     // enum of column jdbc type
     private final int jdbcId;
+    // permitted enum values
+    private final List<String> enumValues;
 
     public VitessType(String name, int jdbcId) {
+        this(name, jdbcId, null);
+    }
+
+    public VitessType(String name, int jdbcId, List<String> enumValues) {
         this.name = name;
         this.jdbcId = jdbcId;
+        this.enumValues = enumValues;
```

Comment:
Can we wrap this into Collections.unmodifiableList(enumValues) here?

Reply:
Good point. Addressed